### PR TITLE
Fix template literal indention

### LINKED ISSUE #13620: Strange indention in template literal
**Prettier 2.7.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEADAOlABDrBGAOiwHEIsYz0QBaagQ2qqwBJhNdcwALOgGwGsCAcwBOcBAQCuUACZwRvAJZQ4ACipcYMAA4BnJAHoDAd1MEo2gLYArXQUiWD2umH50hcJ2K2L5Bl2BwurpUAJTsuAC+ETgATESk5JQgrHTRICxs2BxY3HyCouJQUrLySirqIJo6+kamxuZWtvYQjs6u7p7a3jC+Iv5ggcFhMVjRUKggADQgENq90LrIoHQiIhDGAAqrCEsofMZ0AJ5LMwBGIi78cDAAynSWcAAyynDIAGZ8unDnl643t3ayiEyBgIkkPxAcEsZzgMjkMiedCgQkknQAYhARJY6D4UcgQHRJBRplUYJZeAB1LiKeC6dpwW67WmKAButKOBLAwVJym+Ihgm0uQhxHy+kNsAA9bsDeHAAIqSCDwMW8b4zZwifkEs50WG8UndZQwSmKGQwLjIAAcAAYNetvpTLtoCd0gvJWW8ZgBHJXwIVzPaE3TUFRwuGksS+xRiIXuUVIT5qyHfSyKUHglOyhV+t6J8UzGB603my1IWKFy6KcpCADCrQTUN0AFZSZJvgAVPV7JPqkCsiEASVKsFuYBEinmAEFZLcYEc5arvpFIkA)

```sh
--parser babel
```

**Input:**

```jsx
`
    1. Go to "--a-" ${
      chalk.green.underline("https://www.npmjs.com/package/prettier/access")
    }
    2. Go to "${a}" ${
      chalk.green.underline("https://www.npmjs.com/package/prettier/access")
    }
`
```

**Output:**

```jsx
`
    1. Go to "--a-" ${chalk.green.underline(
      "https://www.npmjs.com/package/prettier/access"
    )}
    2. Go to "${a}" ${chalk.green.underline(
  "https://www.npmjs.com/package/prettier/access"
)}
`;

```

**Expected behavior:**

Those two function calls should format in a similar style.

### PR DESCRIPTION
**Pull request by fisker**
## Description

Fixes #13620

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-js/print/template-literal.js [hunk 1]
```diff
@@ -39,11 +39,11 @@ function printTemplateLiteral(path, print, options) {
   }
   const parts = [];
 
-  let expressions = path.map(print, expressionsKey);
+  let printedExpressions = path.map(print, expressionsKey);
```
Comment (thorn0) [inline src/language-js/print/template-literal.js:42]: How about `expressionDocs`?

Comment (fisker) [reply src/language-js/print/template-literal.js:42]: We always call doc as `printed` in other places.

**Comment by fisker** [comment]
Actually I only changed one line, so I'm going to do a refactor first.

File: src/language-js/print/template-literal.js (hunk 1 above)
Comment (thorn0) [reply src/language-js/print/template-literal.js:42]: Definitely not always.

Comment (fisker) [reply src/language-js/print/template-literal.js:42]: I'm fine with the name, I'll use it.

**Comment by thorn0** [comment]
**Prettier pr-13621**
[Playground link](https://deploy-preview-13621--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEADAOlAxAAgEYA2AhoUTgCTBEC+FwYAFkQQNYB0A5gE5wJsCuUACZwuBAJZQ4ACnQgGMGAAcAzkgD06gO462UJQFsAViraQD6pUTAsiHOJZ6Lxo9QEd+RIV35K4Q5gBpIigOIi4ICAAVLhCVADMILgNROQBKakxMHBycACY2HABxCBwYUrlKRmZ2bl4oAWFRCSlZKFzcuQVlNU0dLT1DEzMICysbOwclJxgXLndPb19-IJCwiOjYqASklK45ABps3Iy5Y5zMXFJSOho6atZOHj5BETFJGS7FVQ1tXX1jKZzJZrLZ7I44M5XB4vD4-AECMFQuFIjE4olkqkQKcoKgQAcQBAlLNoCpkKBwhEtAAFcIIMkoZhaIgATzJBLwsRskIAykQUgAZD7IeLMFRwDlcli88aSDjIGA+CUgOAGPD+ERCAVrTz2ABiuyIzlCyBARH45Xx8hgBgIAHUGOJ4CpxnAefSneIAG5OlmmsAqdkgSTirgwamxDgGIgisXKkwADx5coIcAAivwIPBYwRxQSrFxQ6a8CQ4AQrdNJDA7eIhDAGMgABwABnzEXFdtiSlN0zgoa9cCtHizcAjRIZZpUAFopP5-FaeB5xDwI3Zozm8yBxQZxAqlQSVCn05ns0hRbnlTASDW6w2kHkCYqiOIWhwAMKjdeMgjlgn8cVRCQDLnpuXr8HAACSTSwDyYBcOIxIAILCDyMAsqmG5wNQ1BAA)

```sh
--parser babel
```

**Input:**

```jsx
`
# blabla ${a} ${chalk.green.underline("https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer")}

    2. Go to "${chalk.green.underline(
      "https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer",
    )}"
    
# blabla ${a} ${chalk.green.underline("https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer")}
`
```

**Output:**

```jsx
`
# blabla ${a} ${chalk.green.underline(
  "https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer",
)}

    2. Go to "${chalk.green.underline(
      "https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer",
    )}"
    
# blabla ${a} ${chalk.green.underline(
      "https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer",
    )}
`;

```

File: tests/format/js/template-literals/__snapshots__/jsfmt.spec.js.snap [hunk 2]
```diff
@@ -231,6 +231,168 @@ throw new Error(
...
+          2. Go to "\${chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access",
+          )}" \${chalk.green.underline(
+            "https://www.npmjs.com/package/prettier/access",
+          )}
+  \`,
+  \`
+# blabla \${a} \${chalk.green.underline(
+    "https://www.npmjs.com/package/prettier/quadrupedalKangarooTransformer",
+  )}
```
Comment (thorn0) [inline tests/format/js/template-literals/__snapshots__/jsfmt.spec.js.snap:381]: This indentation looks strange, but it's a separate concern.

Comment (fisker) [reply tests/format/js/template-literals/__snapshots__/jsfmt.spec.js.snap:381]: I agree, the array indention should not apply to the expression.

Comment (fisker) [reply tests/format/js/template-literals/__snapshots__/jsfmt.spec.js.snap:381]: [One leading space makes difference, ](https://deploy-preview-13621--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEw3ADpQNpYAZYDEABAEYA2AhhZcQCTCUC+9wYAFpeQNYB0A5gCc4CXgFcoAEziDyASyhwAFBhDsYMAA4BnJAHo9Ad2O8omgLYArbb0jm9mymG6V+cB8I1yZegI5jKSUExTThJLgBpSih+SkEICAAVQWjtADMIQXMZVQBKJnwsAF0AbiwmCoKoLHRMHEKoYhIaGlZmVg4uPiERKHEpGXlFFTUNHX0jEzMrGzsHJxc3DzgvH39A4NDw8iiYuITk1IysnJB8htLyypAAGhAITRg5aG1kUDj4wwAFOIRXlC4hkoAE9XndSClnCsAMqUbIAGQUcGQaS42jg4Mh3BhjjACn4yBgwQxIDg5lIYWkknh0X4ATcADFMuZKF4YsgQJQxDAILdRuZyAB1dhyeDaXFwaF-UVyABuouBHLA2jBIAU6MEMC+KX4LJRaJJ1gAHtD8eQ4ABFMQQeD68jou6OQQajmkahwch8zSCBQwQVySQwdjIAAcAAZHfF0YKUpoOd64BrZci7v4bXBtQ9-pztABaRRhMJ84T+OTCbWuPVIVH2kno8xyQnEu7aM2W6226sGu4waj+wPBpAAJh7KTkQ34AGEIOYq5zyJ67mJ0YlqP8aw6QLKxHAAJIDWDQsA+x4AQSk0JgwPNdvRFSAA), must caused by [this `LF` check](https://github.com/prettier/prettier/blob/6d6e5177dbb9746e55706f4bece8666df578aceb/src/language-js/print/template-literal.js#L93)

**Review by sosukesuzuki — APPROVED** [review]

**Pull request merged**